src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,46 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrdersByUserId = async (raw_user_id) => {
+  const user_id = +raw_user_id;
```

Comment:
Sorry, but why was this changed again? 
Using the `+` operator to convert between data types is not a good practice.

Reply:
Ow that because it cannot catch when the input is 1r or something with other alphabets 
so I came up with + so it hit the right error, not showing or matching the data

Reply:
I see, you are right. 
In that case, I would recommend using regular expressions instead in the future https://stackoverflow.com/a/19715367